### PR TITLE
Add k4oxygen 

### PR DESCRIPTION
**Pull request by hazel-bunny**
Think I need to update this manually each time Qt gets updated

### CODE REVIEW

File: anda/themes/k4oxygen/k4oxygen.spec [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+%cmake_install
+%undefine _vpath_builddir
+ 
+%global _vpath_builddir %{_target_platform}-qt5
+%cmake_install
+%undefine _vpath_builddir
+ 
+%global _vpath_builddir %{_target_platform}-qt6
+%cmake_install
+%undefine _vpath_builddir
```
Comment (madonuko) [inline anda/themes/k4oxygen/k4oxygen.spec]:
```suggestion
# Build for Qt 4
%global _vpath_builddir %{_target_platform}-qt4
%cmake -DQT_VERSION=4 -B %{_vpath_builddir}
%cmake_build
 
# Build for Qt 5
%global _vpath_builddir %{_target_platform}-qt5
%cmake -DQT_VERSION=5 -B %{_vpath_builddir}
%cmake_build
 
# Build for Qt 6
/usr/bin/patch -p1 -s < %{_sourcedir}/qt6x11_wayland.patch
%global _vpath_builddir %{_target_platform}-qt6
%cmake -DQT_VERSION=6 -B %{_vpath_builddir}
%cmake_build
 
%install
%cmake_install
%undefine _vpath_builddir
 
%cmake_install
%undefine _vpath_builddir
 
%cmake_install
```

File: anda/themes/k4oxygen/k4oxygen.spec [hunk 2]
```diff
@@ -0,0 +1,162 @@
+%global style k4oxygen
+%global dev 10110111
+ 
+%global forgeurl https://github.com/%{dev}/%{style}
+%global commit b0291eff019d4ccd498970d0c8323e49e74a2fec
+%global date 20240329
+%forgemeta
+ 
+Name:    %{style}
+Version: 0
```
Comment (madonuko) [inline anda/themes/k4oxygen/k4oxygen.spec:10]: maybe use `%date.%(c=%commit; echo ${c:0:7})`?

File: anda/themes/k4oxygen/anda.hcl [hunk 3]
```diff
@@ -0,0 +1,6 @@
+project "pkg" {
+    arches = ["x86_64", "aarch64"]
+    rpm {
+        spec = "klassy.spec"
```
Comment (madonuko) [inline anda/themes/k4oxygen/anda.hcl]:
```suggestion
        spec = "k4oxygen.spec"
```

File: anda/themes/k4oxygen/k4oxygen.spec (hunk 2 above)
Comment (hazel-bunny) [reply anda/themes/k4oxygen/k4oxygen.spec:10]: Not interested, in fedora the version is like this: k4oxygen-0.0-1.20240223gitabcdefg.fc40.rpm when I use %date the fedora way. Don't want to break compatibility with that for terra.

File: anda/themes/k4oxygen/k4oxygen.spec [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+%cmake -DQT_VERSION=4 -B %{_vpath_builddir}
+%cmake_build
+ 
+# Build for Qt 5
+%global _vpath_builddir %{_target_platform}-qt5
+%cmake -DQT_VERSION=5 -B %{_vpath_builddir}
+%cmake_build
+ 
+# Build for Qt 6
+/usr/bin/patch -p1 -s < %{_sourcedir}/qt6x11_wayland.patch
```
Comment (madonuko) [inline anda/themes/k4oxygen/k4oxygen.spec:140]:
```suggestion
/usr/bin/patch -p1 -s < %PATCH0
```

File: anda/themes/k4oxygen/k4oxygen.spec [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+%forgemeta
+ 
+Name:    %{style}
+Version: 0
+Release: %autorelease
+Summary: Variant of KDE4 Oxygen widget theme
+License: LGPL-2.1
+URL:     %{forgeurl}
+Source:  %{forgesource}
+Patch:   qt6x11_wayland.patch
```
Comment (madonuko) [inline anda/themes/k4oxygen/k4oxygen.spec:16]:
```suggestion
Patch0:  qt6x11_wayland.patch
```

**Comment by korewaChino** [comment]
pro tip: add a dummy macro that changes every new qt version and have andax update it

**Pull request closed without merging**